__tests__/app.test.js

```diff
@@ -258,3 +258,37 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   })
 });
+
+describe("PATCH /api/articles/:article_id", ()=>{
+  it("Return 201 and updated article", ()=>{
+  const patchObj = {inc_votes:5}
+  return request(app)
+  .patch("/api/articles/1")
+  .send(patchObj)
+  .expect(201)
+  .then((response)=>{
+    expect(response.body[0].votes).toBe(105)
```

Comment:
How do you know your patched the correct article? 
Need to check the article id and all the other values on theobject that comes back.

We should also be getting an object back, not an array